### DISCUSSION TITLE
Setting multiple env variables in build-env

### DISCUSSION THREAD

**Discussion by jacobdo2** [vercel/vercel#5236] [Help] [answered]
According to the documentation here https://vercel.com/docs/cli#commands/overview/unique-options/build-env I should be able to set env variables for build time as follows:

```bash
npx vercel --build-env [NODE_ENV=pr PR_NUMBER=423]
```

but I am getting this odd error:
`Error! The specified file or directory "PR_NUMBER=423]" does not exist.`

**Comment by styfle** [comment, marked as answer]
You would use the flag multiple times if you need multiple environment varaibles.

```
vercel --build-env NODE_ENV=pr --build-env PR_NUMBER=423
```

Or simply

```
vc -b NODE_ENV=pr -b PR_NUMBER=423
```

We'll fix the documentation, thanks!
